src/components/GameView/GameOverDialog.vue

```diff
@@ -90,6 +124,27 @@ export default {
 
       return 'loss-img';
     },
+    currentMatch(){
+      return this.game.currentMatch
+    },
+    matchGameStats(){
+      if(!this.game.currentMatch?.games) return []
+      const result= this.game.currentMatch.games.map(game=>{
```

Comment:
You are optionally chaining the currentMatch variable, do we expect it to be empty? If so, I'd declare it as a variable above, optionally chain it and then use that in the rest of this function. You can even set a default value if we don't need to return early to prevent an extra fork.
```suggestion
      const currentMatchGames = this.game.currentMatch?.games ?? []; 
      const result = currentMatchGames.map((game) => {
```

Reply:
Yeah unranked games won't have matches so it could be empty. This seems a reasonable simplification